static/js/load_map.js

```diff
@@ -75,4 +76,60 @@ function initialize() {
             markers.push(nextMarker);
         });
     });
+
+    const stores = document.getElementById('Stores');
+    function getLocation() {
+        if (navigator.geolocation) {
+            return navigator.geolocation.getCurrentPosition(showPosition);
+        } else {
+            x.innerHTML = "Geolocation is not supported by this browser.";
+        }
+    }
+
+    function showPosition(position) {
+        x.innerHTML = "Latitude: " + position.coords.latitude +
+        "<br>Longitude: " + position.coords.longitude;
+    }
+
+    function callback(results, status) {
+      if (status == google.maps.places.PlacesServiceStatus.OK){
+        markers.forEach(marker => {
+            marker.setMap(null);
+        });
```

Comment:
I haven't seen any style suggestions for when to use `for` vs `forEach`; I've actually been of the impression that `forEach` is for when a data transformation isn't necessarily the goal, whereas `map` is expressly for data transformations. Not entirely sure, though -- I'm not the most familiar with ES5/6 style.